code/Chapter2/Exercise_solution/Fibonacci_test.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <stdexcept>
+#include "gtest/gtest.h"
+
+// Returns the n-th term of the Fibonacci sequence.
+int recursive_fibonacci(int n)
+{
+    if(n < 0)
+        throw std::invalid_argument("Input must be a non-negative number");
+    else if (n <= 1) {
```

Comment:
I'd add braces around this `if` block for consistency with the other branches.

Reply:
Resolved in b8102b89a68c44af785ca07d1b6996a1b5002a7f